bpf/lib/nat.h

```diff
@@ -626,7 +626,7 @@ snat_v4_rev_nat(struct __ctx_buff *ctx, const struct ipv4_nat_target *target,
 		return NAT_PUNT_TO_STACK;
 	};
 
-	if (snat_v4_can_skip(target, &tuple, NAT_DIR_INGRESS, from_endpoint, icmp_echoreply))
+	if (snat_v4_rev_nat_can_skip(target, &tuple))
```

Comment:
I think `icmp_echoreply` is no longer necessary in `snat_v4_rev_nat()`, and can be removed?